### ISSUE TITLE
Cannot enable integration ArchWSL

### ISSUE THREAD

**Issue by ntrp** [docker/for-win#5153]
- [x] I have tried with the latest version of my channel (Stable or Edge)
  - [x] I have uploaded Diagnostics
  - Diagnostics ID: **85485A8C-222B-4FFF-BEA8-ACED8523B1D0/20191115192136**

### Expected behavior

Docker is enabled on the Arch linux distro https://github.com/yuk7/ArchWSL

### Actual behavior
The switch looks fine but once I came back to the settings it was disabled. The following error message is in the log.

```
[20:27:58.997][WslIntegrationAgent-Arch][Info   ] 2019/11/15 20:27:58 Unable to setup docker user group: exit status 127
[20:27:59.003][WslIntegrationAgent-Arch][Warning] Failed to setup docker group in distro, disabling integration
```
### Information

  - Windows Version: **Win 10 Pro 19023.vb_release.191108-1432**
  - Docker Desktop Version: **2.1.5.0**
  - Are you running inside a virtualized Windows e.g. on a cloud server or on a mac VM: **no**

### Steps to reproduce the behavior
Well install a fresh docker for windows (latest edge), install Arch WSL and try to enable the integration. Not sure if it's only my issue.

**Comment by ntrp** [comment]
Additional information, I thought it was related to having a non root user as default but even after reverting to having root as default it gave the same results.

**Comment by ntrp** [comment]
I guess it's a duplicate of this #5107

**Comment by simonferquel** [comment]
Fixed in 2.1.6.0

**Comment by ntrp** [comment]
I can confirm it works fine, thanks, amazing work!
